### PR TITLE
[FIXED JENKINS-40642] Add additional imports for parse/translate

### PR DESCRIPTION
**Pull request by abayer**
[JENKINS-40642](https://issues.jenkins-ci.org/browse/JENKINS-40642)

@NonCPS and @Library were the most glaring missing import, but I
replicated the standard CpsGroovyShell imports - @NonCPS's package,
hudson.model, jenkins.model, and anything provided by a GroovyShellDecorator.

cc @reviewbybees esp @rsandell @kohsuke and @imod

### CODE REVIEW

File: pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/Converter.groovy [hunk 1]
```diff
@@ -108,14 +112,28 @@ public class Converter {
...
 
+    private static CompilerConfiguration makeCompilerConfiguration() {
+        CompilerConfiguration cc = new CompilerConfiguration();
+
+        ImportCustomizer ic = new ImportCustomizer();
+        ic.addStarImports(NonCPS.class.getPackage().getName());
+        ic.addStarImports("hudson.model","jenkins.model");
+        for (GroovyShellDecorator d : GroovyShellDecorator.all()) {
+            d.customizeImports(null, ic);
+        }
```
Comment (imod) [inline pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/Converter.groovy]: I sure must be missing something, but where does `org.jenkinsci.plugins.workflow.libs.Library` get added to the imports?

Comment (abayer) [inline pipeline-model-definition/src/main/groovy/org/jenkinsci/plugins/pipeline/modeldefinition/parser/Converter.groovy]: Through the magic of `GroovyShellDecorator`s - https://github.com/jenkinsci/workflow-cps-global-lib-plugin/blob/master/src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryDecorator.java =)

**Comment by imod** [comment]
👍

**Comment by rsandell** [comment]
non needed :bee: but making the bot happy.

**Pull request merged**